hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/ExitUtil.java

```diff
@@ -204,6 +204,20 @@ public static void resetFirstHaltException() {
     FIRST_HALT_EXCEPTION.set(null);
   }
 
+  /**
+   * Suppresses if legit and returns the first non-null of the two. Legit means
+   * <code>suppressor</code> if neither <code>null</code> nor <code>suppressed</code>.
+   */
```

Comment:
can you add some params to the javadoc to keep it happy

Reply:
done, also added a @return